docs/en/latest/admin-api.md

```diff
@@ -33,6 +34,12 @@ title: Admin API
 - [Plugin Metadata](#plugin-metadata)
 - [Plugin](#plugin)
 
+## Description
+
+The Admin API is a group of APIs served for the Apache APISIX, we could pass parameters to APIs to control APISIX Nodes. To have a better understanding about how it works, please see [the architecture design](./architecture-design/apisix.md).
```

Comment:
What about using term `instances` instead of `nodes`?

Reply:
good good